bedrock/firefox/firefox_details.py

```diff
@@ -46,12 +45,12 @@ class FirefoxDesktop(_ProductDetails):
 
     # Human-readable channel names
     channel_labels = {
-        'nightly': _('Firefox Nightly'),
-        'alpha': _('Developer Edition'),
-        'devedition': _('Developer Edition'),
-        'beta': _('Firefox Beta'),
-        'esr': _('Firefox Extended Support Release'),
-        'release': _('Firefox'),
+        'nightly': 'Firefox Nightly',
```

Comment:
Note: these strings have never been exposed for L10n, as they are mostly product / platform names which we typically display only in English. Because the strings are also used in download buttons, and not just on /firefox/all/, I'm simply going to unwrap them here for now. If we want to translate these in the future, it probably makes sense to add them to a global file, such as `download_button.ftl`.

Reply:
They seem to mostly already be in the global `navigation.ftl` file.